lib/fog/ovirt/requests/compute/v4/update_volume.rb

```diff
@@ -0,0 +1,36 @@
+module Fog
+  module Compute
+    class Ovirt
+      class V4
+        module Shared
+          def check_arguments(id, options)
+            raise ArgumentError, "instance id is a required parameter" unless id
+            raise ArgumentError, "disk id is a required parameter for update-volume" unless options.key? :id
+          end
+        end
+
+        class Real
+          extend ::Fog::Compute::Ovirt::V4::Shared
+
+          def update_volume(id, options)
+            check_arguments(id, options)
+
+            disk_id = options.delete(:id)
```

Comment:
Ditto regarding `.delete` method

Reply:
If we don't want to pass the `:id` later, we should do `options = options.dup` at the beginning`